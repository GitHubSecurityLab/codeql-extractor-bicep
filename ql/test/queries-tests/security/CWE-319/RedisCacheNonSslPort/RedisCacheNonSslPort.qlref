security/CWE-319/RedisCacheNonSslPort.ql
