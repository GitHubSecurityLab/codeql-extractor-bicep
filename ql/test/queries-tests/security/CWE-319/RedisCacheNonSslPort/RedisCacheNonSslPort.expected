| app.bicep:14:1:20:1 | RedisCacheResource | Redis Cache 'insecure-redis' has non-SSL port enabled, which allows unencrypted connections. |
| app.bicep:23:1:29:1 | RedisCacheResource | Redis Cache 'explicit-default-redis' has non-SSL port enabled, which allows unencrypted connections. |
